libutils/fsattrs.h

```diff
@@ -52,6 +52,9 @@ static inline const char *FSAttrsErrorCodeToString(FSAttrsResult res)
         return "File does not exist";
     case FS_ATTRS_NOT_SUPPORTED:
         return "Operation is not supported";
+    default:
+        assert(false && "Bad argument in FSAttrsErrorCodeToString()");
+        return "";
```

Comment:
I'd prefer if we were getting compiler errors if not all values were covered here. But YMMV.

Reply:
So rather return after the switch. I can fix that